### PR TITLE
Add new site

### PR DESCRIPTION
**Pull request by posiaden123**

### CODE REVIEW

File: src/main/java/grabber/sources/librarynovel_com.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
+package grabber.sources;
+
+import com.sun.javadoc.Doc;
```
Comment (Flameish) [inline src/main/java/grabber/sources/librarynovel_com.java:3]: Wrong import

File: src/main/java/grabber/sources/librarynovel_com.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+public class librarynovel_com implements Source{
+    private final String name = "LibraryNovel";
+    private final String url = "https://librarynovel.com/";
+    private final boolean canHeadless = false;
+    private Novel novel;
+    private Document toc;
+
```
Comment (Flameish) [inline src/main/java/grabber/sources/librarynovel_com.java:24]: Missing constructor methods:     
public librarynovel_com() {}

public librarynovel_com(Novel novel) {
    this.novel = novel;
}

File: src/main/java/grabber/sources/librarynovel_com.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                    .cookies(novel.cookies)
+                    .get();
+            Connection.Response res = Jsoup.connect(novel.novelLink)
+                    .cookies(novel.cookies)
+                    .method(Connection.Method.POST)
+                    .data("action","manga_get_chapters")
+                    .data("manga", toc.select("div.post-rating > div.user-rating > input").attr("value"))
+                    .execute();
+            Document doc = res.parse();
+            for (Element chaps : doc.select("div > div > ul > li > a")) {
```
Comment (Flameish) [inline src/main/java/grabber/sources/librarynovel_com.java:54]: Not chapter links

File: src/main/java/grabber/sources/librarynovel_com.java [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        if (toc != null) {
+
+                Element title = toc.selectFirst("div.post-title > h1");
+                Element author = toc.selectFirst("div.author-content > a");
+                Element cover  = toc.selectFirst("div.summary-img > a > img");
+                Element description = toc.selectFirst("div.description-summary > div");
+                metadata.setTitle(title != null ? title.text() : "");
+                metadata.setAuthor(author != null ? author.text() : "");
+                metadata.setDescription(description != null ? description.text() : "");
+                metadata.setBufferedCover(cover != null ? cover.attr("abs:data-src") : "");
```
Comment (Flameish) [inline src/main/java/grabber/sources/librarynovel_com.java:94]: Missing / Wrong metadata

**Review by Flameish — CHANGES_REQUESTED** [review]
I gotta ask, is this like a work in progress because it didn't even work when I tried to test it.

**Comment by posiaden123** [comment]
Yea this was made when I was hella tired, my bad

**Pull request closed without merging**